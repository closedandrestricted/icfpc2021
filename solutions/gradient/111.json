{"vertices":[[101,62],[121,47],[98,131],[88,53],[126,53],[122,74],[67,140],[114,103],[92,84],[98,60],[97,99],[87,116],[71,50],[108,72],[89,86],[71,69],[116,108],[105,128],[112,100],[137,85],[135,71],[64,80],[43,71],[143,108],[120,98],[131,78],[159,136],[166,74],[147,54],[147,54],[133,119],[176,105],[151,97],[165,31],[166,67],[127,152],[141,78],[201,95],[172,116],[157,141],[172,88],[215,64],[196,77],[203,91],[224,34],[192,106],[227,107],[202,61]]}