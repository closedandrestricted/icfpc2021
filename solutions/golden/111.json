{"vertices":[[96,39],[122,38],[87,117],[116,71],[121,43],[108,62],[67,140],[107,92],[89,83],[80,58],[93,99],[88,117],[62,38],[92,68],[107,63],[62,80],[90,87],[101,128],[100,90],[126,103],[120,72],[60,69],[36,71],[114,101],[122,103],[123,72],[158,139],[150,81],[141,49],[146,54],[128,128],[161,83],[149,107],[164,31],[168,62],[133,161],[160,88],[187,73],[173,123],[144,131],[193,103],[207,46],[212,85],[207,75],[211,15],[232,107],[198,102],[223,48]]}